lineman/app/models/group_model.coffee

```diff
@@ -17,6 +17,13 @@ angular.module('loomioApp').factory 'GroupModel', (BaseModel, AppConfig) ->
       @hasMany 'subgroups', from: 'groups', with: 'parentId', of: 'id'
       @belongsTo 'parent', from: 'groups'
 
+    proposals: ->
```

Comment:
I tried this and it didn't work, @robguthrie can shed some light as to why. We had to do it manually in the end.